From 561f242048e4d2cc3d712fd5a0e76435e6c622e7 Mon Sep 17 00:00:00 2001
From: meka rama <v.meka@samsung.com>
Date: Sat, 15 Oct 2016 05:22:26 +0900
Subject: [PATCH] weston: libinput-seat: enable weston backend even if no input
 devices attached

This scenario is required to use HDMI displays without connecting LCD displays.
usually, HDMI displays have no touch input device. So, weston should allow such
scenario.

Signed-off-by: meka rama <v.meka@samsung.com>
---
 src/libinput-seat.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libinput-seat.c b/src/libinput-seat.c
index 5168890..b45f36d 100644
--- a/src/libinput-seat.c
+++ b/src/libinput-seat.c
@@ -260,10 +260,10 @@ udev_input_enable(struct udev_input *input)
 			"warning: no input devices on entering Weston. "
 			"Possible causes:\n"
 			"\t- no permissions to read /dev/input/event*\n"
+			"\t- no Input Device is attached\n"
 			"\t- seats misconfigured "
 			"(Weston backend option 'seat', "
 			"udev device property ID_SEAT)\n");
-		return -1;
 	}
 
 	return 0;
-- 
2.1.4

